lld/ELF/SyntheticSections.cpp

```diff
@@ -2327,8 +2327,9 @@ SymtabShndxSection::SymtabShndxSection()
 
 void SymtabShndxSection::writeTo(uint8_t *buf) {
   // We write an array of 32 bit values, where each value has 1:1 association
-  // with an entry in .symtab. If the corresponding entry contains SHN_XINDEX,
-  // we need to write actual index, otherwise, we must write SHN_UNDEF(0).
+  // with an entry in .ctx.symtab-> If the corresponding entry contains
```

Comment:
Is this comment change intentional. The code looks to be referring to the .symtab section ctx.in.symTab.